### ISSUE TITLE
source of param 'state' issue after  pip install python-linkedin

### ISSUE THREAD

**Issue by ntlilianfeng** [ozgur/python-linkedin#96]
In the github, the source code of  authorization_url state is:
'state': self.state or self._make_new_state(),

But, after pip install python-linkedin, the source code of authorization_url state is:
self.state = self._make_new_state()
'state': self.state,

which caused the param 'state' cann't be assign a value.

**Comment by thundergolfer** [comment]
@ntlilianfeng check the date of the package on [pypi](https://pypi.python.org/pypi/python-linkedin/4.1), it's not up to do date with the current repo source code. For one thing it doesn't support Python 3.

**Comment by iamthesiz** [comment]
Yeah. Same problem here. Honestly though, it's kind of ridiculous that linkedin doesn't manage oauth libraries themselves...
